src/Cassandra.IntegrationTests/Core/MetadataTests.cs

```diff
@@ -899,5 +899,43 @@ public void GetMaterializedView_Should_Refresh_View_Metadata_Via_Events()
                 Assert.Null(cluster.Metadata.GetMaterializedView("ks_view_meta2", "monthlyhigh"));
             }
         }
+        /// Tests that clustering order metadata is set properly
+        /// 
+        /// ColumnClusteringOrderReversedTest tests that clustering order metadata for a column is properly recalled in the driver
+        /// metadata under the "clustering_order" column. It first creates a simple table with a primary key, one column ascending, another
+        /// column descending, and the final column without any order. It checks the metadata for each column to make sure that the proper
+        /// value is recalled in the driver metadata.
+        /// 
+        /// @since 3.0.0
+        /// @jira_ticket CSHARP-359
+        /// @expected_result Clustering order metadata is properly set
+        /// 
+        /// @test_category metadata
+        [Test, TestCassandraVersion(3, 0)]
+        public void ColumnClusteringOrderReversedTest()
+        {
+            string keyspaceName = TestUtils.GetUniqueKeyspaceName();
+            string tableName = "products";
+            ITestCluster testCluster = TestClusterManager.GetNonShareableTestCluster(DefaultNodeCount);
+            var cluster = testCluster.Cluster;
+            var session = testCluster.Session;
+            session.CreateKeyspace(keyspaceName);
+            session.ChangeKeyspace(keyspaceName);
+
+            var cql = "CREATE TABLE " + tableName + " (" +
+                      @"id int,
+                      description text,
+                      price double,
+                      tags frozen<set<text>>,
+                      PRIMARY KEY(id, description, price)
+                      ) WITH COMPACT STORAGE
+                      AND CLUSTERING ORDER BY (description ASC, price DESC)";
+            session.Execute(cql);
+
+            var tableMeta = cluster.Metadata.GetKeyspace(keyspaceName).GetTableMetadata(tableName);
+            Assert.IsFalse(tableMeta.ColumnsByName["description"].IsReversed);
```

Comment:
`IsReversed` is an internal property used for C* < 3.0, not part of the public API.
You should use `ClusteringKeys` that is an array of tuples containing the column and the clustering order, something like:
```csharp
CollectionAssert.AreEqual(new[] { "description", "price" }, table.ClusteringKeys.Select(c => c.Item1.Name));
CollectionAssert.AreEqual(new[] { SortOrder.Ascending, SortOrder.Descending }, table.ClusteringKeys.Select(c => c.Item2));
```